lib/generators/sufia/templates/workflow.json.erb

```diff
@@ -33,6 +59,10 @@
                     "methods": [
                         "CurationConcerns::Workflow::ActivateObject"
                     ]
+                }, {
+                    "name": "depositor_updated",
+                    "from_states": [{"names": ["depositor_review"], "roles": ["depositor_updated"]}],
```

Comment:
Change `from_state` name to `changes_requested`.

Change role to `depositing`. See above question re: making sure we add this role to all registered users (or all users who can deposit to an admin set, if this workflow is used in the context of an admin set).